.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - abhijeetnarvekar
+  - package-ecosystem: "nuget"
+    directory: "/"
```

Comment:
is this directory here correct? Or does it have to be "/Acrolinx.Net.Demo/", @abhijeetnarvekar ?

Reply:
@Thilo-Acrolinx I am a bit confused. 

In this case /Acrolinx.Net.Demo/ makes sense.
Every .NET projects generally have there own separate directory.

But if I want to add a test project, it would be another directory. Acrolinx.Net.Demo.Test.

Does depedabot automatically scan for all directories inside the repo?  If so, we can just keep it `/`

Reply:
@Thilo-Acrolinx Yes it needs to be /Acrolinx.Net.Demo/

Reply:
Changed, please review again @abhijeetnarvekar.